src/llm_utils.py

```diff
@@ -54,18 +54,13 @@ def analyze_code_changes(diff_content: str) -> List[Dict]:
         model="gemini-2.0-flash", contents=prompt
     )   
 
-    #print(response.text)
+    print(response.text)
     
     # Parse and format the response
```

Comment:
The commented-out block is unnecessary. Remove it.

```suggestion
Remove commented section
```